docs/mindsdb-docs/docs/sql/tutorials/diabetes.md

```diff
@@ -0,0 +1,290 @@
+<div style="float:right;" ><img src="/assets/tutorials/diabetes/pg4admin/images.png" width="200" height="150" /></div><div> <img src="/assets/tutorials/diabetes/pg4admin/diabetes_logo.png" style="float:left;" width="50" height="50" /><h1><strong>Diabetes</strong></h1></div>
+
+*Dataset:[Diabetes Data](https://github.com/mindsdb/mindsdb-examples/blob/a43f66f0250c460c0c4a0793baa941307b09c9f2/others/diabetes_example/dataset/diabetes-train.csv)*
+
```

Comment:
Can you add yourself as an author, please 

```
Communtiy Author: your_name
```

Reply:
amended